platforms/Windows/bld/bld.wxs

```diff
@@ -0,0 +1,351 @@
+﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui">
+  <Package
+      Language="1033"
+      Manufacturer="swift.org"
+      Name="!(loc.Bld_ProductName)"
+      UpgradeCode="7e95dc06-7f84-4e8e-a038-8304af0468fb"
+      Version="$(var.ProductVersion)"
+      Scope="$(PackageScope)">
+
+    <MediaTemplate CabinetTemplate="bld{0}.cab" />
+
+    <DirectoryRef Id="_usr_include">
+      <Directory Id="_usr_include_llvm_c" Name="llvm-c" />
+    </DirectoryRef>
+
+    <DirectoryRef Id="_usr_lib_swift">
+      <Directory Id="_usr_lib_swift_swiftToCxx" Name="swiftToCxx" />
+      <Directory Id="_usr_lib_swift_migrator" Name="migrator" />
+    </DirectoryRef>
+
+    <DirectoryRef Id="_usr">
+      <Directory Name="share">
+        <Directory Id="_usr_share_swift" Name="swift" />
+      </Directory>
+    </DirectoryRef>
+
+    <ComponentGroup Id="binutils" Directory="_usr_bin">
+      <!-- TODO(compnerd) can we use symbolic links to llvm-ar.exe instead? -->
```

Comment:
Would be great to know if you have an answer to this question - is there a way to use a symbolic link that can be laid down rather than duplicating the binary itself.